routers/routes/routes.go

```diff
@@ -523,6 +523,11 @@ func RegisterRoutes(m *macaron.Macaron) {
 				return
 			}
 		})
+		m.Group("/branches", func() {
```

Comment:
If you have a tab, you should define a new unit in `unit.go` and update `repo_unit`  table.